server/config/environment/production.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
             process.env.PORT ||
             8080,
 
+  // Server Address
+  addr:     process.env.SERVER_ADDR ||
+            "",
+
+  // Sendgrid email
+  sendgridApiKey: process.env.SENDGRID_API_KEY,
```

Comment:
so actually this variable should be defined for production but doesn't have to be defined for development, which is why development falls back to empty string but production doesn't fall back